exo-deploy/src/deploy.ls

```diff
@@ -3,6 +3,10 @@ require! {
   'require-yaml'
 }
 
+command-flag = process.argv[2]
```

Comment:
The CLI does parse the command-line arguments, but then forwards it as an argument when building the `docker run ... #{image-name} #{command-flag}` string. That will start `exo-deploy` inside a container with a command flag. So this line is actually reading `process.argv[2]` from inside the docker container not from the user's command line.